app/controllers/admin/users_controller.rb

```diff
@@ -58,15 +63,19 @@ def sort_whitelist
   end
 
   def filter_whitelist
-    %w[username email]
+    %w[username email group]
   end
 
-  def filter
+  def filter(rel) # rubocop:disable Metrics/AbcSize
```

Comment:
is there a specific reason why these two filter functions look so vastly different?

Reply:
The generic one generates a query like:

`SELECT * FROM groups WHERE name = "something"`, but the User's filter can do either:

`SELECT * FROM users WHERE username ILIKE '%something%'` OR, it can do the more complicated:
`SELECT * FROM users LEFT JOIN users_groups ON users_groups.user_id = users.id WHERE users_groups.group_id = 'some-group-id'`